pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = ["ansys-dpf-core>=0.7.2",
                 "ansys-platform-instancemanagement~=1.0",
                 "pyvista>=0.43.4",
                 "hollerith==0.4.1",
-                "numpy>=2.0.0",

```

Comment:
@RobPasMue should we pin a low version of numpy based on their support lifecycle? how do we do this in other libraries?

Reply:
That would be a good idea, indeed. But currently there's no standard AFAIK. I can have a look though.